packages/core/admin/webpack.config.js

```diff
@@ -81,7 +81,16 @@ module.exports = ({
         {
           test: /\.tsx?$/,
           loader: require.resolve('esbuild-loader'),
-          include: [cacheDir, ...pluginsPath],
+          include: pluginsPath,
```

Comment:
This wasn't quite what I meant, what I meant was how we have it earlier but edit the `exclude` to dynamically not include the `pluginPaths` list of strings that were inside node_modules.

Reply:
I think you need to keep the `cacheDir` unless you exclude it and always use `babel-loader`. This part is a little bit tricky and not the best one IMO, we have to to keep babel-loader because we developed a custom babel plugin to make the distinction between the ee and the ce versions. However, esbuild was a request from the community as it should be faster than ts-loader, honestly I haven' checked the difference between the 2 it could be worth investigating as it would make the config easier to maintain.

Reply:
> I think you need to keep the cacheDir unless you exclude it and always use babel-loader.

I'm not suggesting we remove this. Right now the PR is not the right way we've discussed internally.